docs/Configuration/configuration-files/README.md

```diff
@@ -838,7 +838,9 @@ The interval at which to check for webhook conditions. This value currently conf
   webhook_settings:
     interval: "12h"
   ```
-
+  > **Note:** Fleet webhook policies default to a 24h time interval based upon the initial start time of the policy. To adjust policy automation intervals, set the interval to a longer period and manually trigger automations using fleetctl. You'll see small differences over time as well based on how long it takes to run, other jobs that are queued up at the same time, etc.
```

Comment:
Great! Very clear.  Good use of imperative mood on second sentence, starting with “why/what” and then “how”.

One nit: I’d open the first sentence like this:

> Fleet’s webhook notifications about failing policies…


To be totally clear what we mean, especially for folks who might be less familiar with all features